tools/run_danesfield.py

```diff
@@ -230,17 +230,18 @@ def main(args):
     #############################################
 
     input_paths = []
-    for root, dirs, files in itertools.chain(os.walk(config['paths']['imagery_dir']),
-                                             os.walk(config['paths']['rpc_dir'])):
+    for root, dirs, files in itertools.chain(os.walk(config['paths']['imagery_dir'])
+                                             #os.walk(config['paths']['rpc_dir'])
```

Comment:
I think we could have a flag defined above based on whether or not `'rpc_dir'` is in `config['paths']`, and exclude it from this call based on that